app/scripts/styles/_uswds-theme.scss

```diff
@@ -14,6 +16,12 @@
         "mobile-lg": false,
         "desktop": false
     ),
-    $theme-font-type-sans: baseFontFamily,
     $theme-font-type-serif: baseFontFamily,
-);
\ No newline at end of file
+    $theme-font-type-sans: 'public-sans',
```

Comment:
move declaration to `theme.ts` as a new root variable, or change the existing font over there and keep `baseFontFamily`